concordium-consensus/src/Concordium/Skov/Update.hs

```diff
@@ -221,6 +241,10 @@ processFinalization newFinBlock finRec@FinalizationRecord{..} = do
                     _ -> return prunedbrs
         newBranches <- trimBranches unTrimmedBranches
         putBranches newBranches
+        -- mark dead blocks by increasing height
+        forM_ (toRemoveFromTrunk ++ reverse toRemoveFromBranches) $ \bp -> do
```

Comment:
I think they should be removed by decreasing height to avoid having a live block with a dead parent.

Reply:
I think at this point it should not matter, but I don't have any objections to it either.